src/components/dragelement/index.js

```diff
@@ -140,8 +140,7 @@ dragElement.init = function init(options) {
         if(options.doneFn) options.doneFn(gd._dragged, numClicks, e);
 
         if(!gd._dragged) {
-            var e2 = document.createEvent('MouseEvents');
-            e2.initEvent('click', true, true);
+            var e2 = new MouseEvent('click', e);
```

Comment:
Oh. I think that will break click in IE9 and even up to IE11.

I'd be ok with something like:

```js
var e2

try {
  e2 = new MouseEvent('click', e);
} catch(e) {
  e2  = document.createEvent('MouseEvents');
  e2.initEvent('click', true, true);
}
```

and hence only passing `MouseEvent` in browsers that supports it.